modules/styling/stories/GettingStarted.stories.mdx

```diff
@@ -0,0 +1,89 @@
+<Meta title="Features/Styling/Getting Started" />
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-styling
+```
+
+## Usage
+
+```tsx
+import React from 'react';
+import {createRoot} from 'react-dom/client';
+
+import {cs} from '@workday/canvas-kit-styling';
+
+const myStyles = createStyles({
+  backgroundColor: 'red',
+}); // returns the CSS class name created for this style
+
+myStyles; // something like "css-{hash}"
+
+const domNode = document.getElementById('root');
+const root = createRoot(domNode);
+
+root.render(<div className={myStyles}>Hello!</div>);
+```
+
+## Development
+
+Canvas Kit Styling comes with a runtime that doesn't need anything special for developement. The
+runtime uses `@emotion/css` to include your styles on the page.
+
+## Production
+
+If you wish to use the static compilation, you must use the `@workday/canvas-kit-styling-parser`
```

Comment:
is this still part of this PR? the parser?

Reply:
Not part of this PR, but another PR will add it. It is difficult to maintain 2 separate PRs and fix this type of thing. I'd rather leave this alone for now.